docs/docs/command-line-interface.mdx

```diff
@@ -279,11 +279,31 @@ You can start the converter by running:
 rasa data convert nlu
 ```
 
-You can specify the input file, output file, and the output format with the following arguments:
+You can specify the input file/directory, output file/directory, and the output format with the following arguments:
 
 ```text [rasa data convert nlu --help]
 ```
 
+## rasa data convert core
+
+You can convert Core data from Markdown to YAML.
+
+You can specify the input file/directory, output directory with the following arguments:
+
+```bash
+rasa data convert core --help
+```
+
+## rasa data convert nlg
+
+You can convert NLG data from Markdown to YAML.
```

Comment:
what is NLG data? Do you mean the responses.md for the response selector?

Reply:
also can you add `nlg` to the word exclusion list for typo CI please

Reply:
> what is NLG data? Do you mean the responses.md for the response selector?
Yes! mentioned that in the docs above

> also can you add nlg to the word exclusion list for typo CI please
✅

Reply:
hmmm, i would still explicitly mention that here. and did we officially decide we're calling responses.md NLG now? i can also start that discussion on slack though, i'm gonna be re-reviewing this page later anyways

Reply:
What I've added is:

> You can convert existing NLU, Stories, and NLG (i.e. `responses.md`) training data

Let's indeed discuss that on Slack, I'm happy to rename NLG to whatever we decide on. Currently NLG is a term we use in the code itself.